models/gc_grt123_lung_cancer/meta.json

```diff
@@ -0,0 +1,171 @@
+{
+  "id": "2e67a3cc-4680-4058-bf4e-f965cf50f06f",
+  "name": "lung_cancer_risk_estimation",
```

Comment:
must match the model name: `gc_grt123_lung_cancer`

Reply:
NOTE: I updated our automated checks to prevent a model name mismatch from slipping through.

Reply:
The name has been matched.